query-engine/connectors/mongodb-query-connector/src/root_queries/mod.rs

```diff
@@ -38,3 +41,17 @@ fn pick_singular_id(model: &ModelRef) -> ScalarFieldRef {
         .next()
         .unwrap()
 }
+
+pub(crate) async fn metrics<'a, F, T, U>(f: F) -> mongodb::error::Result<T>
+where
+    F: FnOnce() -> U + 'a,
+    U: Future<Output = mongodb::error::Result<T>>,
+{
+    let start = Instant::now();
+    let res = f().await;
+
+    histogram!("query_total_elapsed_time_ms", start.elapsed());
```

Comment:
Is this name shared with the sql connector? It may be a good idea to move the strings to constants and put them somewhere shared (either connector interface or a new crate, ...).

Reply:
It is the same. Sadly its set in quaint. So I cannot share it.